### PR TITLE
gcsfuse 0.1.1 (new formula)

### PR DESCRIPTION
**Pull request by jacobsa**
For GoogleCloudPlatform/gcsfuse#63.

### CODE REVIEW

File: Library/Formula/gcsfuse.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+class Gcsfuse < Formula
+  desc "A user-space file system for interacting with Google Cloud Storage"
+  homepage "https://github.com/googlecloudplatform/gcsfuse"
+  url "https://github.com/GoogleCloudPlatform/gcsfuse/archive/v0.1.0.tar.gz"
+  sha256 "a65a1a70f75361f7ee9339199be98318d933b7986648860e179b5f24b130b156"
+
+  depends_on :osxfuse
+  depends_on "go" => :build
+
+  # Declare versioned Go dependencies
```
Comment (mistydemeo) [inline Library/Formula/gcsfuse.rb]: We don't need comments on standard things like this - would you mind removing them? Thanks!

**Comment by jacobsa** [comment]
I removed the line you attached a comment to. I've left the comments in the install section because as a new formula author I had a lot of trouble understanding what was going on with other Go formulas. But if you intend for those to be removed too, let me know and I will.

**Comment by mikemcquaid** [comment]
> I removed the line you attached a comment to. I've left the comments in the install section because as a new formula author I had a lot of trouble understanding what was going on with other Go formulas. But if you intend for those to be removed too, let me know and I will.

Yeh, that'd be good, thanks.

**Comment by jacobsa** [comment]
I've cut down the comments; please take another look.

File: Library/Formula/gcsfuse.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    bin.install "gcsfuse.bin" => "gcsfuse"
+    bin.install "gcsfuse_mount_helper.bin" => "gcsfuse_mount_helper"
+  end
+
+  test do
+    # We don't have GCS credentials lying around, and probably don't want to
+    # cause a file system to be mounted. So just make sure the programs were
+    # built.
+    system "ls", bin/"gcsfuse", bin/"gcsfuse_mount_helper"
```
Comment (mikemcquaid) [inline Library/Formula/gcsfuse.rb]: Need more than just an `ls` here. Can we run `--version` or `--help` or something (as that's better than nothing).

Comment (jacobsa) [inline Library/Formula/gcsfuse.rb]: Done. (Had to update to v0.0.1 for this, too.)

**Comment by jacobsa** [comment]
`--help` tests added, please take another look.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**